docs/REST API/rest-api.md

```diff
@@ -9057,10 +9057,10 @@ Content-Type: application/octet-stream
 
 _Available in Fleet Premium._
 
-Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
+Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts. Changes to the installer package will reset
+installation counts. Changes to any field other than `self_service` will cancel pending installs for the old package.
```

Comment:
I'd keep this description simple and move this information to a note below the parameters table.
```suggestion
Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
```